backends/bmv2/simple_switch/simpleSwitch.cpp

```diff
@@ -803,6 +803,36 @@ void ExternConverter_action_selector::convertExternInstance(
     ctxt->action_profiles->append(action_profile);
 }
 
+namespace {
```

Comment:
LGMT. Only a question here. Why is this extra namespace necessary?

Reply:
This prevents the linker from creating this as a global symbol. We've had bugs in the past where local passes (no header files) clashed between files. These cause extremely difficult to diagnose bugs (runtime crashes). The C++ linker does not complain, and is non-deterministic.